src/Propel/Generator/Util/QuickBuilder.php

```diff
@@ -426,6 +427,10 @@ public function buildClasses(?array $classTargets = null): void
         foreach ($includes as $tempFile) {
             include($tempFile);
         }
+
+        if (in_array('tablemap', $classes)) {
```

Comment:
we should set 3rd arg to true here for stringish compares.

Reply:
ok